thermo_nerf/evaluator/evaluator.py

```diff
@@ -64,6 +64,10 @@ def _compute_metrics(
             camera_ray_bundle,
```

Comment:
Can you update the variable name to represent the correct type? Maybe `cameras` instead of `camera_ray_bundle`? Thanks a lot for fixing this issue.

I just realized that you updated thermoNeRF to the latest Nerfstudio version also in some sense so I appreciate that :)

Reply:
@MalcolmMielle I rushed to share the code I was using to solve a problem on my own, and in doing so, I made some basic mistakes. I know it’s not easy to review everything thoroughly, so I really appreciate your kind and detailed feedback. This is my first time collaborating with international colleagues, so I’m still getting used to it. Could you please confirm if it’s okay for me to push the new commit like this? :)

Reply:
Hej! No problem it is really excellent to have you help out on this project! We also release this code a little quickly and apparently some error slipped in so it's really nice of you to help fix them. Pushing the commit this way is great, we will squash the PR at the end to have only one commit :).

For my comment here, I'm under the impression that the variable `camera_ray_bundle` line 64, should be the one named `cameras` while the one line 69 should remained `camera_ray_bundle` since it's the actual bundle of rays. Once this is changed, I'll merge it

Reply:
I've just pushed the changes. Thanks again for taking the time to review everything; I really appreciate it.